### DISCUSSION TITLE
Question about reinitialise the Postgres container database

### DISCUSSION THREAD

**Discussion by melvillewong** [melvillewong/SSH#27] [Q&A]
I am trying to reinitialize the migrations, starting from the original 3 versions to a new V1__initial_ssh_smart_scheduling.sql.

1. Am I supposed to drop the whole database and recreate it?

I've forgotten the process for doing this.
I deleted the flyway_schema_history table, but an error occurred saying that some types had already been created. It seems that the database retained the original content for other tables.

**Comment by melvillewong** [comment]
I found out that I need to drop the entire database and recreate it in order to apply the new migration script.

Thanks to @chapwiz and @anson224 .

**Reply by anson224** [reply]
The conversation we had is certainly contributive to the development. 

@chapwiz @melvillewong
